src/components/ChartContainer.js

```diff
@@ -12,13 +12,12 @@ import html2canvas from "html2canvas";
 import jsPDF from "jspdf";
 import ChartNode from "./ChartNode";
 import "./ChartContainer.css";
+import { TransformWrapper, TransformComponent } from "react-zoom-pan-pinch";
 
 const propTypes = {
   datasource: PropTypes.object.isRequired,
   pan: PropTypes.bool,
-  zoom: PropTypes.bool,
-  zoomoutLimit: PropTypes.number,
-  zoominLimit: PropTypes.number,
+  zoom: PropTypes.object,
```

Comment:
We should be able to define the shape of the props of the zoom object.

```
PropTypes.shape({
    prop1: PropTypes.string,
    prop2: PropTypes.number
  }),
```

Reply:
Oh how I miss typescript! Thanks for the suggestion and I'll push up an update

Reply:
🔥 

Do you want to see what it'd take to just drop `tsc init` in there? We don't need to do backward support, just new code?

Reply:
I'd be down to do this in a separate ticket but not related to the actual zoom functionality

Reply:
👍🏼